app/pages/organization/stats/components/BarChart/BarChartContainer.jsx

```diff
@@ -61,15 +61,15 @@ class BarChartContainer extends React.Component {
     const query = {
       period: binBy,
       projectID,
-      type
+      type: type + 's'
```

Comment:
As opposed to replace `type` from `classification` to `classifications` (and `comment` to `comments`) on OrganizationStats.jsx references to `<BarChart>`, I opted to add the plural form when creating the query to ERAS. 

I ran into translations errors if I changed `classification` to `classifications` for Org Stats. 
For eg. I would have to update for all locales `organization.stats.classification` to `organization.stats.classifications` (same thing for `organization.stats.comment` and `organization.stats.perclassification` etc)...

Let me know if that didnt make sense.

Reply:
100% understand approach and works 👍 . However, just in case we forget or someone in future gets confused and with my familiarity with organization code 🙈 I opened https://github.com/zooniverse/Panoptes-Front-End/pull/6986 - let me know what you think.